mne/io/edf/tests/test_edf.py

```diff
@@ -123,6 +125,12 @@ def test_edf_data():
                            stim_channel=None, exclude=['Ergo-Left', 'H10'],
                            verbose='error')
     raw_py = read_raw_edf(edf_path, stim_channel='auto', preload=True)
+
+    # Test original units
```

Comment:
as in brainvision if we load the file in anyother place without using `_test_raw_reader` i would factor it out, use a fixutre and create a **unit** test for the units. Otherwise, add it to the common tests in `_test_raw_reader` but I think that the later wont be easy.

Reply:
what's the argument against having the test here? It does not require additional loading, simply accessing an existing attribute and making two logical comparisons

Reply:
That brainvision has a `test_original_units()` and edf does not. To me, all readers should have similar tests and structured in the same manner. Then if the argument of just sneaking couple asserts here with a comment `# Test original units` instead of defining a function `def test_original_utings()` is to avoid overhead, we should start fixtures. Even if they are pretty fast.